proposals/0540-vision-for-layout-conformance-parity.md

```diff
@@ -0,0 +1,238 @@
+
+## Example capabilities
+
+The following is a non-exhaustive list of capabilities where Yoga lags browsers. We should consult specs used by yoga (CSS Box Alignment, CSS Box Sizing, CSS Flexbox, etc) for a more exhaustive list.
+
+1. **`display: grid`:** CSS Grid is highly used in responsive designs on the modern web, allowing some layouts which are hard or not possible to express with Flexbox. It is one of the most requested additions, but has one of the highest implementation cost of the listed examples.
+2. **`display: inline/block/table/ruby`:** Yoga treats text as inline, but other nodes act as `display: flex`. The listed display modes could help compatibility, but Flexbox has superseded much of their usage.
+3. **`display: contents`:** Useful for composing boxes which should not alter child layout (e.g. embedding event handlers)`.`
+4. **`position: fixed/sticky`:** React native does not support `position: fixed` or `position: sticky`. React Native’s closest API is `stickyHeaderIndices`, which allows top-level ScrollView items to be sticky.
+5. **`box-sizing`:** React Native does not support specifying box-sizing, and anecdotally is closest to but does not fully conform to `border-box`. implementing box-sizing allows new modes which may be conformant.
+6. **`align-content: space-evenly`:** Yoga does not supported `space-evenly` as a value for flexbox alignment.
+7. **`place-content`:** Shorthand for `align-content` and `justify-content`. Trivial to add but has a memory const without [sparse stylesheets](#mitigating-bloat-to-heap-usage).
+8. **`inset:`** Shortland for top/left/bottom/right (similar considerations to `place-contents` as a shorthand property)
+9. **Viewport based units `(vmin, vh, vw, etc)`:** React Native + Yoga does not support units derived from viewport/root node size.
+10. **Font based units `(em, rem, etc)`:** React Native + Yoga does not support units derived from the size of the current font. Configurable `fontSize`  in React Native is currently confined to text components, and will not cascade. Its pixel value would then be scaled by `fontSize` if set on a Text component or by system default font if unset on Text, or on a View.
+11. **Percentages in more places:** Some existing functionality like gap is implemented with support for pixels, but are missing support for percentage.
+12. **`min-content`:** As part of not supporting content-based minimum sizing, the `min-content` keyword is not supported.
```

Comment:
I suspect this isn't worth adding unless we add min/max sizing for flex basis. It's almost always used for min/max sizes - I've literally never seen it used outside of those two properties

Reply:
It would probably make sense to add support for `max-content` and `fit-content` if this is being added.